_sass/vendor/pygments/index.html

```diff
@@ -0,0 +1,858 @@
+<!DOCTYPE html>
```

Comment:
No, I think we can remove this file. At the time, I thought it might be useful when browsing the theme docs locally, but I now think the theme website should merely link to the relevant page in the Pygments website.